packages/flutter_feed_utils/lib/src/widgets/overview_layout.dart

```diff
@@ -0,0 +1,81 @@
+import "package:flutter/cupertino.dart";
+import "package:flutter/material.dart";
+import "package:flutter/widgets.dart";
+
+enum _OverviewLayoutOption {
```

Comment:
make this public, the options should be dynamically determined by the consuming side. Ideally we want to have an easy way to programmatically switch between layout options